higlass_manage.py

```diff
@@ -661,7 +661,8 @@ def browse(names):
         print("Error: higlass instance not found. Have you tried starting it using 'higlass-manage start'?", file=sys.stderr)
         return
 
-    webbrowser.open('http://localhost:{port}/app/'.format(port=port))
+    if not os.environ.get('CI'):
```

Comment:
Maybe use a more obscure environment variable name
```suggestion
    # make sure this test passes on Travis CI and doesn't try to open
    # a terminal-based browser which doesn't return
    if not os.environ.get('HAS_JOSH_K_SEAL_OF_APPROVAL'):
```